src/instrumenter/instrument.ts

```diff
@@ -530,7 +557,14 @@ export function generateExpressions(
     // Step 1: Define struct holding all the temporary variables neccessary
     const exprs = annotations.map((annot) => annot.expression);
     const factory = ctx.factory;
-    const structName = uid.get("vars");
+    const allNames = getAllNames(contract);
+
+    let possibleStructName = uid.get("vars");
+    while (allNames.has(possibleStructName)) {
```

Comment:
We can use `getNewVarName(possibleStructNames, "vars")` here instead of the while loop.